src/jit/compiler.cpp

```diff
@@ -2321,58 +2321,42 @@ const char* Compiler::compLocalVarName(unsigned varNum, unsigned offs)
 #ifdef _TARGET_XARCH_
 static bool configEnableISA(InstructionSet isa)
 {
-#ifdef DEBUG
     switch (isa)
     {
         case InstructionSet_SSE:
             return JitConfig.EnableSSE() != 0;
         case InstructionSet_SSE2:
-            return JitConfig.EnableSSE2() != 0;
+            return JitConfig.EnableSSE() != 0 && JitConfig.EnableSSE2() != 0;
```

Comment:
I think, for the common case, most of these would be more efficient if we had the latest ISA check first. -- In production scenarios, users generally won't be disabling ISAs and we can cut out the majority of the checks if the hardware doesn't have any support for the target ISA in general.

Reply:
Good point, will change.